src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -17,7 +25,31 @@ void checkSpecificAccessControl() {
 
     @Override
     JsonResult execute() {
-        // TODO: implement the logic for creating a feedback session log
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String fsName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        FeedbackSessionAttributes feedbackSession = getNonNullFeedbackSession(fsName, courseId);
```

Comment:
No need to make a db query to fetch feedback session here.. just log whatever is hit. We do not want to waste time trying to do heavy validation.

Reply:
Noted! Removed the validations